flocker_bb/builders/flocker.py

```diff
@@ -659,12 +660,25 @@ def getBuilders(slavenames):
 ]
 
 
+def build_automatically(branch):
+    """
+    :param branch: The name of a branch.
+
+    Return a bool, whether the branch should be built after a push without
+    being forced.
+    """
+    return branch == 'master' or branch.startswith('release/')
```

Comment:
`flocker_bb.steps` has a `BranchType` set of constants and a `getBranchType` helper. It'd be better to use those rather than duplicating the heuristic here.